querySQL/MinioSQL.ipynb

```diff
@@ -17,7 +17,19 @@
     "*Note: Amazon S3 Select does not support whole-object compression for Parquet objects.*\n",
     "[Source](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Client.select_object_content)\n",
     "\n",
-    "**NOTE: The examples here use JSON, but CSV is better suited to large datasets, performing 10x faster in my experiment.**"
+    "**NOTE: The examples here use JSON, but CSV is better suited to large datasets, performing 10x faster in my experiment.**\n",
+    "\n",
+    "**NOTE: The .Parquet example is commented out. Parquet is DISABLED by default since hostile crafted input can easily crash the server.**\n",
```

Comment:
Add this note as well near the example.  good chance people scroll straight to the example (as I did not) and not see the note